core/extension/gdextension_interface.h

```diff
@@ -2126,6 +2126,19 @@ typedef void (*GDExtensionInterfaceRefSetObject)(GDExtensionRefPtr p_ref, GDExte
  */
 typedef GDExtensionScriptInstancePtr (*GDExtensionInterfaceScriptInstanceCreate)(const GDExtensionScriptInstanceInfo *p_info, GDExtensionScriptInstanceDataPtr p_instance_data);
 
+/**
+ * @name object_get_script_instance
+ * @since 4.2
+ * 
+ * Get the script instance data attached to this object.
+ * 
+ * @param p_object A pointer to the Object
```

Comment:
```suggestion
 * @param p_object A pointer to the Object.
```
End with period, same on the two lines below